javascripts/govuk/anchor-buttons.js

```diff
@@ -0,0 +1,55 @@
+(function(global, settings) {
+  "use strict"; 
+       
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};  
+
+  GOVUK.anchorButtons = {
+    
+    // default configuration that can be overridden by passing object as second parameter to module
+    config: $.extend({
+      // setting to true will disable the init function upon execution
+      disableInit: false,
+      // the target element(s) to attach the shim event to
+      selector: 'a[role="button"]',
```

Comment:
Can we add this role to other elements such as spans?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_button_role

Maybe the selector should be just `[role="button"]`.